app/content-scripts/content_script_bundle.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Cliqz Content Scripts
+ *
+ * This file compiles content scripts for Cliqz modules
+ * (Ad-Blocker, HPN, Human Web)
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+/**
+ * @namespace CliqzContentScript
+ */
+
+import 'browser-core/build/core/content';
```

Comment:
Shouldn't this be including `browser-core/build/core/content-script.bundle` instead?